CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## v0.18.4 (2017-11-16)
+
+### Enhancements
+
+#### :gear: WebSockets support ([testcafe-hammerhead/#911](https://github.com/DevExpress/testcafe-hammerhead/issues/911))
+TestCafe now provides full-featured WebSockets support (`wss` and `ws` protocols, request authentication, etc.). 
+
+### Bug Fixes
+* TestCafe can now click on elements that are located under the Status bar and have the `transition` css property specified ([#1934](https://github.com/DevExpress/testcafe/issues/1934))
+* Added support for pages with the `rest` and `default parameter` instructions ([testcafe-hammerhead/#1336](https://github.com/DevExpress/testcafe-hammerhead/issues/1336))
+* Pages with several `base` tags are supported now ([testcafe-hammerhead/#1349](https://github.com/DevExpress/testcafe-hammerhead/issues/1349))
+* Redirects from cross-domain to same-domain pages are processed now ([testcafe-hammerhead/#1363](https://github.com/DevExpress/testcafe-hammerhead/issues/1363))
```

Comment:
This issue is still opened in the repository. Should we close it or remove it from the changelog?
/cc @miherlosev @LavrovArtem

Reply:
We've are fixed the partial case. The problem covers whole url processing mechanism.
This is why we don't close an appropriate issue.